src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -329,6 +336,8 @@ export default class QueryEditorUI extends Component<Props, State> {
         return monaco.languages.CompletionItemKind.Field;
       case 'value':
         return monaco.languages.CompletionItemKind.Value;
+      case 'keyword':
```

Comment:
this is what i said, it is now returning monaco type directly with the new change and this is being removed. [PR](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7441)

Reply:
will updates completion item kind type in fast follow